Content.Shared/Movement/Pulling/Systems/PullingSystem.cs

```diff
@@ -248,6 +248,12 @@ private void OnPullableMoveInput(EntityUid uid, PullableComponent component, ref
         if (!_blocker.CanMove(entity))
             return;
 
+        if (args.OldMovement == MoveButtons.None)
```

Comment:
```suggestion
        if (args.OldMovement != MoveButtons.None)
```
Isn't this the wrong way around?
I tested it and now keydown won't break the pull.
Keydown means switching from none to a direction.